motr/client.h

```diff
@@ -641,10 +641,15 @@ enum m0_entity_type {
 	 *   m0_entity_open(NULL, &obj->ob_entity, &ops[0]);
 	 */
 	M0_ENF_META = 1 << 0,
+	/**
+	 * If this flags is set during entity_create() that means application
+	 * do not support update operation. This flag is not in use yet.
```

Comment:
The comment says 'This flag is not in use yet'. Do you want to remove it?

Reply:
I think re-phrasing it would be better, as it is used by S3 team, but the flag has no implementation, so it has not effect as of now.